### ISSUE TITLE
Release RGE-GS models on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [CN-ADLab/RGE-GS#1]
Hi @CN-ADLab 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work on Arxiv and your paper "RGE-GS: Reward-Guided Expansive Driving Scene Reconstruction via Diffusion Priors" got featured on our paper page: https://huggingface.co/papers/2506.22800.
The paper page lets people discuss about your paper and lets them find artifacts about it (your models for instance), you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

I saw in your abstract that your source code will be made publicly available on your GitHub repository (https://github.com/CN-ADLab/RGE-GS). It would be fantastic if you would consider hosting the pre-trained models associated with RGE-GS on https://huggingface.co/models!

Hosting your models on Hugging Face will give them more visibility and enable better discoverability within the AI community. We can add relevant tags in the model cards (e.g., `image-to-3d` for driving scene reconstruction) so that people can easily find and use your work, and link it directly to your paper page.

If you're interested, here's a guide on uploading models: https://huggingface.co/docs/hub/models-uploading. If it's a custom PyTorch model, the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class simplifies adding `from_pretrained` and `push_to_hub` functionalities. Alternatively, users can download files directly using [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file).

After uploading, we can also help link the models to your paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) for seamless discovery.

You could also build a demo for your model on [Spaces](https://huggingface.co/spaces), and we can potentially offer a ZeroGPU [grant](https://huggingface.co/docs/hub/en/spaces-gpus#community-gpu-grants) to provide A100 GPUs for free.

Please let me know if you're interested or need any guidance!

Kind regards,

Niels
